### PR TITLE
Webview start

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
@amuramoto And the web view start

### CODE REVIEW

File: webview/app.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+ */
+
+'use strict';
+
+// Imports dependencies and set up http server
+const
+    request = require('request'),
+    express = require('express'),
+    body_parser = require('body-parser'),
+    crypto = require('crypto'),
```
Comment (amuramoto) [inline webview/app.js]: crypto is removed from the final, but is still present in the starter.

File: webview/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Webview Screen - Starter Project
+
+This is a starter project you can use to as a starting point for completing the [Messenger Platform Webview tutorial](#).
```
Comment (amuramoto) [inline webview/README.md]: add link

File: webview/app.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    }
+});
+
+
+// Handles messages events
+function handleMessage(sender_psid, received_message) {
+
+}
+
+function setRoomPreferences(sender_psid) {
```
Comment (amuramoto) [inline webview/app.js]: Add a comment describing the stub

File: webview/app.js [hunk 4]
```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright 2017-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ * Starter Project for Messenger Platform Quick Start Tutorial
```
Comment (amuramoto) [inline webview/app.js]: Update this description. This isn't for the quick start

**Comment by ChrisChinchilla** [comment]
@amuramoto All updated

File: webview/app.js [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+app.set('port', process.env.PORT || 5000);
+app.use(body_parser.json());
+app.use(express.static('public'));
+
+const PAGE_ACCESS_TOKEN = process.env.PAGE_ACCESS_TOKEN;
+const SERVER_URL = process.env.SERVER_URL;
+const APP_SECRET = process.env.APP_SECRET;
+
+app.listen(app.get('port'), function () {
```
Comment (amuramoto) [inline webview/app.js]: @ChrisChinchilla convert to arrow function

File: webview/app.js [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+app.post('/webhook', (req, res) => {
+
+    // Parse the request body from the POST
+    let body = req.body;
+
+    // Check the webhook event is from a Page subscription
+    if (body.object === 'page') {
+
+        body.entry.forEach(function (entry) {
```
Comment (amuramoto) [inline webview/app.js]: @ChrisChinchilla convert to arrow function

**Pull request merged**